stable_baselines/gail/dataset/dataset.py

```diff
@@ -29,10 +30,12 @@ class ExpertDataset(object):
         the data (slower but use less memory for the CI)
     """
 
-    def __init__(self, expert_path, train_fraction=0.7, batch_size=64,
-                 traj_limitation=-1, randomize=True, verbose=1,
-                 sequential_preprocessing=False):
-        traj_data = np.load(expert_path)
+    def __init__(self, expert_path=None, traj_data=None, train_fraction=0.7, batch_size=64,
+                 traj_limitation=-1, randomize=True, verbose=1, sequential_preprocessing=False):
+        if traj_data is not None and expert_path is not None:
+            raise ValueError("Cannot specify both 'traj_data' and 'expert_path'")
+        if traj_data is None:
+            traj_data = np.load(expert_path)
```

Comment:
you should also check that expert_path is not None, no?

Reply:
My logic here was that we weren't previously checking for this, and `np.load` would generate an error. However on reflection I agree now I've made `expert_path` a keyword rather than positional argument I should check for this; added.